config.env.test

```diff
@@ -27,4 +23,10 @@ BN_UPDATER_SP_ADDRESS_IDCHAIN=0x183C5D2d1E43A3aCC8a977023796996f8AFd2327
 BN_UPDATER_MAINNET_WSS=
 BN_UPDATER_IDCHAIN_WSS=wss://idchain.one/ws/
 BN_UPDATER_SEED_GROUPS_WS_URL=wss://idchain.one/ws/
-BN_ARANGO_EXTRA_OPTS=
\ No newline at end of file
+BN_ARANGO_EXTRA_OPTS=
+# passwords
+SEED=
```

Comment:
add BN_ in front of this.

Reply:
https://github.com/BrightID/BrightID-Node/pull/286/commits/7ed87f6a4dbc721a55e4b89ba2a17089c4b9c8f1